testsuite/features/step_definitions/vm_steps.rb

```diff
@@ -13,27 +13,28 @@
   cloudinit_image = "/var/testsuite-data/cloudinit-disk-#{os_type}.iso"
   memory = '512'
 
-  mac = ENV.fetch('MAC_MIN_NESTED', 'RANDOM')
-  STDOUT.puts 'Random MAC address used' if mac == 'RANDOM'
-
   case os_type
   when 'leap'
     name = 'leap-disk-image-template.qcow2'
     net = 'salt-leap'
     os = 'opensuse15.4'
     memory = '1024'
+    mac = 'RANDOM'
```

Comment:
Move `mac = 'RANDOM'` above the case statement, to reduce some lines.

Reply:
Yes, makes sense. Thank you!